facturacion_mexico/facturacion_fiscal/tests/test_derive_pac_reconciliation.py

```
@@ -0,0 +1,211 @@
+
+	def test_valid_cancellation_desconocida(self):
+		self._assert("valid", "no_existe", None, SyncStates.ERROR)
+
+	# --- normalización ---
+	def test_normaliza_mayusculas_y_espacios(self):
+		self._assert("  CANCELED ", "", FiscalStates.CANCELADO, SyncStates.SYNCED)
+		self._assert("Valid", "EXPIRED", FiscalStates.TIMBRADO, SyncStates.SYNCED)
+		self._assert(None, None, None, SyncStates.ERROR)
+
+
+def _seed_si(*, fiscal_status="", ffm=None):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_seed_si`

(ANN202)